Last-Update: 2014-05-21
Forwarded: https://github.com/ceph/ceph/pull/1844
Bug-Debian: http://bugs.debian.org/748571
Author: John David Anglin <dave.anglin@bell.net>
Description: Define AO_REQUIRE_CAS to fix FTBFS on 'hppa'.

--- a/src/include/atomic.h
+++ b/src/include/atomic.h
@@ -25,6 +25,7 @@
 #ifndef NO_ATOMIC_OPS
 
 // libatomic_ops implementation
+#define AO_REQUIRE_CAS
 #include <atomic_ops.h>
 
 // reinclude our assert to clobber the system one
